### ISSUE TITLE
Asking About Training Code

### ISSUE THREAD

**Issue by 11w22334** [skrantidatta/LIPINC#2]
Hello, your code has been very helpful to me. I would like to start learning your work from the training code. May I ask when the training code will be available? The testing code does not include details about the loss function and training parameters. Thank you!

**Comment by skrantidatta** [comment]
Our team has decided not to release the training code. However, the loss function and training parameters can be found in the model.py file (lines 149-173) and in Section IV of the paper, under "Experiment Settings."
![image](https://github.com/user-attachments/assets/79c39a80-d660-4b1d-ae4b-69ff96f58895)
